src/ConfigCatProvider.tsx

```diff
@@ -104,6 +108,33 @@ function serverContextNotSupported(): Error {
 }
 
 class ConfigCatClientStub implements IConfigCatClient {
+  getValue<T extends SettingValue>(_key: string, _defaultValue: T, _callback: (value: SettingTypeOf<T>) => void, _user?: User): void {
+    throw serverContextNotSupported();
+  }
+  getValueDetails<T extends SettingValue>(_key: string, _defaultValue: T, _callback: (evaluationDetails: IEvaluationDetails<SettingTypeOf<T>>) => void, _user?: User): void {
+    throw serverContextNotSupported();
+  }
+  forceRefresh(_callback: (result: RefreshResult) => void): void {
+    throw serverContextNotSupported();
+  }
+  getAllKeys(_callback: (value: string[]) => void): void {
+    throw serverContextNotSupported();
+  }
+  getAllVariationIds(_callback: (variationIds: string[]) => void, _user?: User): void {
+    throw serverContextNotSupported();
+  }
+  getAllVariationIdsAsync(_user?: User): Promise<string[]> {
+    throw serverContextNotSupported();
+  }
+  getKeyAndValue(_variationId: string, _callback: (settingkeyAndValue: SettingKeyValue<SettingValue> | null) => void): void {
+    throw serverContextNotSupported();
+  }
+  getAllValues(_callback: (result: SettingKeyValue<SettingValue>[]) => void, _user?: User): void {
+    throw serverContextNotSupported();
+  }
+  getAllValueDetails(_callback: (result: IEvaluationDetails<SettingValue>[]) => void, _user?: User): void {
+    throw serverContextNotSupported();
+  }
```

Comment:
These methods are unnecessary as they were removed in [configcat-common v8.0.0](https://github.com/configcat/common-js/releases/tag/v8.0.0). I guess you have an outdated version of configcat-common in `node_modules` of your working copy.

```suggestion
```

Reply:
thanks the note, the common lib was updated and the interface methods too.